app/pubspec.yaml

```diff
@@ -70,6 +70,8 @@ dependencies:
   get_time_ago: ^1.1.5
   implicitly_animated_reorderable_list: ^0.4.2
   cached_memory_image: ^1.3.1
+  flutter_mentions: ^2.0.1
+  flutter_html: ^3.0.0-alpha.5
```

Comment:
@gnunicorn @sachintha-s  Okay I've checked their compatibility issues with ```flutter_svg```. Apparently the dev is still using old version library of ```flutter_svg``` in the package which kinda sucks despite of many community requests. We can go with it but its alpha stage and many features missing and its not recommended for production. The workaround is to downgrade ```flutter_svg``` which also I would not recommend doing it.

Reply:
Just for heads-up as we will upgrade this soon as they add latest version of ```flutter_svg```.